src/LookbookEmbed.jsx

```diff
@@ -0,0 +1,121 @@
+import React, {useState} from 'react'
+import {ActionMenu, ActionList, Link, Box} from '@primer/react'
+
+const baseUrls = {
+  rails: 'https://primer.style/view-components/lookbook/preview/primer/'
+}
+
+// https://primer.style/view-components/lookbook/preview/primer/beta/button/playground/
+
+export function LookbookEmbed({framework = 'rails', status, component, previews}) {
+  const [colorScheme, setColorScheme] = useState('light')
+  const base = baseUrls[framework]
+  const [preview, setPreview] = React.useState(Object.keys(previews)[0])
+  const statusString = `${status}`
+  const componentString = `${component}`
+  const previewString = `${preview}`
+  const options = [statusString, componentString, previewString].join('/')
+  console.log(options)
+
+  const url = `${base}${new URLSearchParams(options)}`
```

Comment:
JavaScript friends, can anyone tell me why `URLSearchParams` is adding a `=` to the end of the url? This seems to work to populate the iframe without the `=` 😄 

CC @jonrohan @broccolinisoup

Reply:
Hi @langermank  👋🏼  I think it does it because `URLSearchParams` work with query strings and query strings are like key/value format in URLs like `baseURL?key1=value1&key2=value2` etc . See [the mdn ref](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) for examples. 

I think here the options are more like query params than query strings. (`beta/button/playground`) so I guess we don't need `URLSearchParams` here unlike we do in StorybookEmbed. We can just pass the options without creating a new `URLSearchParams` object.

Reply:
Thank you @broccolinisoup! I didn't realize `URLSearchParams` was expecting a key/value pair 😆 just hacked my way through it. Your suggestion works perfectly! 🙌